ocw_data_parser/ocw_data_parser_test.py

```diff
@@ -663,6 +665,34 @@ def test_extract_media_locally(ocw_parser):
     assert counts == expected_counts
 
 
+def test_populate_vtt_files(ocw_parser):
+    """populate_vtt_files should duplicate srt content files"""
+    subrip_count = 0
+    srt_json = {}
+    with open(
+        "ocw_data_parser/test_json/course_dir/captions_example.json", "rb"
+    ) as file:
+        datafield = json.load(file)
+        for loaded_json in ocw_parser.jsons:
+            if loaded_json["_content_type"] == "application/x-subrip":
+                loaded_json["_datafield_file"] = datafield
+                subrip_count += 1
+                if subrip_count == 1:
+                    srt_json = loaded_json
+    files_count_before = len(ocw_parser.jsons)
+    ocw_parser.populate_vtt_files()
+    assert files_count_before + subrip_count == len(ocw_parser.jsons)
+
+    vtt_file_id = update_srt_to_vtt(srt_json["id"])
+    vtt_json = next(
+        (new_json for new_json in ocw_parser.jsons if new_json["id"] == vtt_file_id),
+        None,
+    )
+    assert sys.getsizeof(srt_json["_datafield_file"]) == sys.getsizeof(
```

Comment:
Can we assert the number of bytes in `vtt_json["_datafield_file"]` instead? `sys.getsizeof` gets the number of bytes used in memory which isn't quite the same thing